<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OffsetCalibrationWidget</class>
 <widget class="QWidget" name="OffsetCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Offset calibration</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsTabWidget">
      <attribute name="title">
       <string>Scan</string>
      </attribute>
      <layout class="QFormLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox">
         <property name="title">
          <string>Optics</string>
         </property>
         <layout class="QFormLayout">
          <item row="0" column="0">
           <widget class="QListWidget" name="quadsListWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPlainTextEdit" name="opticsEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel">
            <property name="text">
             <string>Import from file:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDialogButtonBox" name="saveButtonBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Open|QDialogButtonBox::Save</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel">
            <property name="text">
             <string>Import from focus:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QFormLayout">
            <item row="1" column="1">
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLineEdit" name="focusEdit">
                <property name="text">
                 <string>1,2,3,4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="focusButton">
                <property name="text">
                 <string>Read</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel">
         <property name="text">
          <string>Number of shots:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="numUsedSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel">
         <property name="text">
          <string>Save raw data:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLineEdit" name="fileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fileButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="statusTabWidget">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPlainTextEdit" name="logEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsTabWidget">
      <attribute name="title">
       <string>Results</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="TableView" name="resultsTable">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
